### ISSUE TITLE
Issue: License has expired

### ISSUE THREAD

**Issue by vmpnable** [web-pal/chronos-timetracker#206]
Hi, my team and I have been using the Chronos Timetracker power-up on Trello. In the power-up details page, it made no mention about pricing, so we thought it was free; however, today, we're unable to use the power-up because it says "Chronos License has expired!" We can't seem to find the link for pricing on Trello, but I've found [this link](https://marketplace.atlassian.com/apps/1216398/chronos-timetracking-system-for-jira?hosting=cloud&tab=pricing) to the pricing plugin for JIRA.

If we have to pay for the Trello power-up, how do we do so? Just want to ask the community/developer for advice on this as there doesn't seem to be anything online about using it on Trello.

**Comment by GeorgeXCV** [comment]
Agree, loved the plugin until this happened. If you want to charge, fair enough. But the way it is presented on Trello makes it look like a free plugin, wasn't aware I was on a trial either.
